joebvp/makevoigt.py

```diff
@@ -95,18 +97,27 @@ def get_lsfs():
             cfg.lsfs.append(lsf['kernel'])
             break
         else:
-
-            lamobs=np.median(cfg.wave[fg])
+            try:
+                lamobs=np.median(cfg.wave[fg])
+            except:
+                import pdb; pdb.set_trace()
```

Comment:
clean this?

Reply:
Lots of debugging involved in getting the LSF issues worked out...cleaning this stuff up now!